.circleci/config.yml

```diff
@@ -200,16 +200,26 @@ jobs:
             - run: sudo chown -R circleci:circleci /usr/local/lib/python3.7/site-packages
             - restore_cache:
                   key: deps9-{{ .Branch }}-{{ .Environment.CIRCLE_SHA1 }}
+            - run:
+                  command: |
+                      cd python-packages/json_schemas
+                      python -m ensurepip
+                      python -m pip install .[dev]
+                      # HACK! installing the package should do the following
+                      # copy for us, but it's not working in CircleCI for some
+                      # reason.  Support ticket raised (#43979).
```

Comment:
Can you give us a full URL to the ticket?

Reply:
I see, can you add the fact that it's a Zendesk ticket? I was going to start search on Github.